microsoft-edge/webview2/release-notes/index.md

```diff
@@ -0,0 +1,1318 @@
+---
+title: Release Notes for the WebView2 SDK
+description: Release notes for Microsoft Edge WebView2, covering new features, APIs, and fixes for Win32, WPF, and WinForms.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 03/18/2024
+---
+# Release Notes for the WebView2 SDK
+
+These Release Notes provide information about new features and non-security updates that are included in the Microsoft Edge Stable Channel.
+<!-- todo: 
+is that true for the WebView2 Prerelease SDK, or only for the WebView2 Release SDK?
+mention "WebView2", "Runtime", & "SDK"?
+-->
+
+See also:
+* [Release notes for Microsoft Edge Security Updates](/deployedge/microsoft-edge-relnotes-security)
+* [Archived release notes for Microsoft Edge Stable Channel](/deployedge/microsoft-edge-relnote-archive-stable-channel)
+* [Overview of the Microsoft Edge channels](/deployedge/microsoft-edge-channels)
```

Comment:
I must have not communicated properly with the first comment. I meant that we should have a similar section to the Edge release notes, but not the exact same links.

For our case, it should just be:
* [About Release Notes for the WebView2 SDK](./about.md)
* [Archived Release Notes for the WebView2 SDK](./archive.md)

Reply:
Instead, in the 3 articles, in See Also section at bottom, cross-linked.  Added lead-in sentence in the 3 articles.